src/DIRAC/DataManagementSystem/Client/FTS3Job.py

```diff
@@ -224,6 +224,28 @@ def monitor(self, context=None, ftsServer=None, ucert=None):
 
         return S_OK(filesStatus)
 
+    def cancel(self, context):
+        """Cancel the job on the FTS server. Note that it will cancel all the files.
+        See https://fts3-docs.web.cern.ch/fts3-docs/fts-rest/docs/api.html#delete-jobsjobidlist
+        for behavior details
+        """
+
+        try:
+            cancelDict = fts3.cancel(context, self.ftsGUID)
+            newStatus = cancelDict["job_state"].capitalize()
```

Comment:
```suggestion
            newStatus = cancelDict["job_state"].upper()
```
```
>>> 'Cancel'.capitalize()
'Cancel'
>>> 'CANCEL'.capitalize()
'Cancel'
>>> 'CANCEL'.upper()
'CANCEL'
>>> 'Cancel'.upper()
'CANCEL'
>>> 
```

Reply:
oops well done ! I will change that, and be consistent (so use `Cancel` and not `CANCEL`)